### PR TITLE
[Docs] Japanese translation of docs/reference_info_json.md

### PR DESCRIPTION
**Pull request by umi-umi**
Add a Japanese translation of docs/reference_info_json.md.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@shelaf @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

File: docs/ja/reference_info_json.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+* `key_count`
+   * **必須**: このレイアウトのキーの数
+* `layout`
+   * 物理レイアウトを説明するキー辞書のリスト。詳細は次のセクションを見てください。
+
+### キー辞書形式
+
+レイアウトの各キー辞書は、キーの物理プロパティを記述します。<http://keyboard-layout-editor.com> の Raw Code に精通している場合、多くの概念が同じであることが分かります。可能な限り同じキー名とレイアウトの選択を再利用しますが、keyboard-layout-editor とは異なって各キーはステートレスで、前のキーからプロパティを継承しません。
+
+全てのキーの位置と回転は、キーボードの左上と、各キーの左上からの相対で指定されます。
```
Comment (shelaf) [inline docs/ja/reference_info_json.md:47]: in relation toは、～との関係ということで相対と訳されたのかなと思いますが、ここの内容からも相対的な位置についてはあまり述べられていないことから、基準と訳すのはどうでしょうか
```suggestion
全てのキーの位置と回転は、キーボードの左上と、各キーの左上を基準にして指定されます。
```

File: docs/ja/reference_info_json.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+* `y`
+   * **必須**: 垂直軸でのキーの絶対位置(キー単位)。
+* `w`
+   * キー単位でのキーの幅。`ks` が指定された場合は無視されます。デフォルト: `1`
+* `h`
+   * キー単位でのキーの高さ。`ks` が指定された場合は無視されます。デフォルト: `1`
+* `r`
+   * キーを回転させる時計回りの角度。
+* `rx`
+   * 水平軸内でキーを回転させる場所の絶対位置。デフォルト: `x`
```
Comment (shelaf) [inline docs/ja/reference_info_json.md:60]:
```suggestion
   * キーを回転させる点の水平軸における絶対位置。デフォルト: `x`
```

File: docs/ja/reference_info_json.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+* `ry`
+   * 垂直軸内でキーを回転させる場所の絶対位置。デフォルト: `y`
```
Comment (shelaf) [inline docs/ja/reference_info_json.md:62]:
```suggestion
   * キーを回転させる点の垂直軸における絶対位置。デフォルト: `y`
```

File: docs/ja/reference_info_json.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+* `ks`
+   * キー形状: キー単位で場所のリストを提供することでポリゴンを定義します。
```
Comment (shelaf) [inline docs/ja/reference_info_json.md:64]:
```suggestion
   * キー形状: キー単位で点のリストを提供することでポリゴンを定義します。
```

File: docs/ja/reference_info_json.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+   * **重要**: これらはキーの左上からの相対で、絶対ではありません。
+   * ISO Enter の例: `[ [0,0], [1.5,0], [1.5,2], [0.25,2], [0.25,1], [0,1], [0,0] ]`
+* `label`
+   * マトリックス内のこの位置につける名前。
+   * これは通常PCB上でこの位置にシルクスクリーン印刷されるものと同じ名前でなければなりません。
```
Comment (shelaf) [inline docs/ja/reference_info_json.md:69]:
```suggestion
   * これは通常 PCB 上でこの位置にシルクスクリーン印刷されるものと同じ名前でなければなりません。
```

File: docs/ja/reference_info_json.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+   * キー単位でのキーの高さ。`ks` が指定された場合は無視されます。デフォルト: `1`
+* `r`
+   * キーを回転させる時計回りの角度。
+* `rx`
+   * 水平軸内でキーを回転させる場所の絶対位置。デフォルト: `x`
+* `ry`
+   * 垂直軸内でキーを回転させる場所の絶対位置。デフォルト: `y`
+* `ks`
+   * キー形状: キー単位で場所のリストを提供することでポリゴンを定義します。
+   * **重要**: これらはキーの左上からの相対で、絶対ではありません。
```
Comment (shelaf) [inline docs/ja/reference_info_json.md:65]:
```suggestion
   * **重要**: これらはキーの左上からの相対位置で、絶対位置ではありません。
```

File: docs/ja/reference_info_json.md (hunk 1 above)
Comment (umi-umi) [reply docs/ja/reference_info_json.md]: ご指摘ありがとうございます。
いきなり 相対・絶対という言葉が出てきて、うーんどうなのかなと思っていましたが、
基準 という言葉を使うとすっきりしますね。
ありがとうございます！

**Comment by noroadsleft** [comment]
@mtei Looks good to me, but merge it whenever you think it's ready.

File: docs/ja/reference_info_json.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+* `h`
+   * キー単位でのキーの高さ。`ks` が指定された場合は無視されます。デフォルト: `1`
+* `r`
+   * キーを回転させる時計回りの角度。
+* `rx`
+   * キーを回転させる点の水平軸における絶対位置。デフォルト: `x`
+* `ry`
+   * キーを回転させる点の垂直軸における絶対位置。デフォルト: `y`
+* `ks`
+   * キー形状: キー単位で点のリストを提供することでポリゴンを定義します。
```
Comment (mtei) [inline docs/ja/reference_info_json.md:64]:
```suggestion
   * キー形状: キー単位で頂点を列挙することでポリゴンを定義します。
```

**Review by mtei — APPROVED** [review]

**Comment by mtei** [comment]
Thank you!

**Pull request merged**